scripts/test-utils/test-fixture-dir.ts

```diff
@@ -132,7 +132,20 @@ export function testFixtureDir<T extends TestFixtureConfig>(
 
             for (const [outputName, content] of Object.entries(outputs)) {
                 const outputPath = path.resolve(dirname, outputName);
-                expect(content).toMatchFile(outputPath);
+
+                try {
+                    if (content === undefined) {
+                        expect(fs.existsSync(outputPath)).toBe(false);
+                    } else {
+                        await expect(content).toMatchFileSnapshot(outputPath);
+                    }
+                } catch (err) {
+                    if (typeof err === 'object' && err !== null) {
+                        Error.captureStackTrace(err, testFixtureDir);
+                    }
+
+                    throw err;
+                }
```

Comment:
If the content is undefined, toMatchFileSnapshot would create a file with the `undefined` as its content, so we check if the file doesn't exist instead.

Catching the error and capturing the stack trace makes sure the failure will be reported in the spec file instead of `test-fixture-dir.ts` .

Reply:
```suggestion
                try {
                    await expect(content ?? '').toMatchFileSnapshot(outputPath);
                } catch (err) {
                    if (typeof err === 'object' && err !== null) {
                        Error.captureStackTrace(err, testFixtureDir);
                    }

                    throw err;
                }
```
We previously generated just one `error` or `expected` file with content, and removed the other. We found that generating both, and leaving one empty, made it easier to work with the diffs. If I understand correctly, then using `''` as a fallback for `undefined` should preserve the existing behavior, rather than going back to conditionally writing files.

Reply:
I applied this suggestion. The snapshots were updated now with the empty files. Is this correct?

Reply:
Wait, there shouldn't be any new snapshots added for just a tweak to the test util. I guess I was wrong about where we do the handling for the empty files. Let's just go with your original implementation. Sorry about that! 😓

Reply:
No problem. Reverted the change.